control/wifi_new.proto

```diff
@@ -142,3 +142,22 @@ message ScanNetworksReply {
 
   repeated Network networks = 1; // All found networks
 }
+
+/**
+ * Store network credential but do not attempt to power on wifi or connect.
+ *
+ * On success, the network credentials get saved to a persistent storage.
+ * On failure, there was not enough room to store the credentials
```

Comment:
This can only happen if DVOS runs out of memory when resizing the 10 item credentials vector. I dont think this is likely to happen?

Do we want to make it explicit that if the device has 10 credentials and we send another one that the 10th credential will be dropped? 
Do we want to change this behavior?

Reply:
Probably an edge case?

Reply:
Ideally, we would want to replace SSIDs which are old / least recently connected. But that's probably an overkill? What we can do though is have another control request that loads the config file and then display the SSIDs it in the CLI.